pkg/vm/precompiles.go

```diff
@@ -154,12 +174,13 @@ func getPrecompiles(config params.ChainConfig, app *application.Avalanche) (para
 		nativeMint        = "Native Minting"
 		contractAllowList = "Contract deployment whitelist"
 		txAllowList       = "Transaction allow list"
+		feeManager        = "Manage Fee Settings"
```

Comment:
This list looks a bit odd, there is a mix of all first case uppercase and lowercase

Reply:
good catch, cleaned up